lib/ApiPlatform/Serializer/InvalidSearchConditionNormalizer.php

```diff
@@ -27,53 +31,80 @@
 {
     public const FORMAT = 'jsonproblem';
 
-    /**
-     * @var UrlGeneratorInterface
-     */
-    private $urlGenerator;
+    private $serializePayloadFields;
+    private $nameConverter;
 
-    public function __construct(UrlGeneratorInterface $urlGenerator)
+    public function __construct(array $serializePayloadFields = null, NameConverterInterface $nameConverter = null)
     {
-        $this->urlGenerator = $urlGenerator;
+        $this->nameConverter = $nameConverter;
+        $this->serializePayloadFields = $serializePayloadFields;
     }
 
     /**
-     * Normalizes an object into a set of arrays/scalars.
-     *
-     * @param InvalidSearchConditionException $object  object to normalize
-     * @param string                          $format  format the normalization result will be encoded as
-     * @param array                           $context Context options for the normalizer
-     *
-     * @return array
+     * @inheritdoc
```

Comment:
No need for `@inheritdoc`, this is implicit and only applies to the description.